packages/agreement-process/test/agreementService.integration.test.ts

```diff
@@ -1181,4 +1182,24 @@ describe("Agreement service", () => {
       );
     });
   });
+  describe("get agreement", () => {
+    it("should get an agreement", async () => {
+      const agreement: Agreement = buildAgreement(
+        generateId<EServiceId>(),
+        generateId<TenantId>(),
+        agreementState.draft
+      );
+      await addOneAgreement(agreement, postgresDB, agreements);
+
+      const result = await agreementService.getAgreementById(agreement.id);
+      expect(result).toEqual(agreement);
```

Comment:
For the happy path: could we have 2 agreements in the readmodel? So that we are sure that the id filter is actually working.

Reply:
Makes sense, done!